### PR TITLE
Reflected Aqua's converter refactoring

### PR DESCRIPTION
**Pull request by a-matsuo**
### Summary
Reflected Aqua's converter refactoring 

### Details and comments
Some methods names in tutorials were changed according to https://github.com/Qiskit/qiskit-aqua/pull/1061

### CODE REVIEW

File: tutorials/optimization/2_converters_for_quadratic_programs.ipynb [hunk 1]
```diff
@@ -371,7 +257,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Call encode method of `LinearEqualityToPenalty` to convert."
+    "Call `conevrt` method of `LinearEqualityToPenalty` to convert."
```
Comment (Cryoris) [inline tutorials/optimization/2_converters_for_quadratic_programs.ipynb:260]:
```suggestion
    "Call `convert` method of `LinearEqualityToPenalty` to convert."
```

**Comment by Cryoris** [comment]
The PR that this was depending on in Aqua has been merged, could you merge the current master branch and trigger the tests @nonhermitian or @a-matsuo?

**Comment by nonhermitian** [comment]
This works on released qiskit versions. So will need to wait for release to verify.

**Comment by Cryoris** [comment]
Ah alright, thanks! 👍

**Comment by a-matsuo** [comment]
@nonhermitian @Cryoris I ran notebooks on the latest version, and no errors occurred. Could you please check this PR?

**Comment by nonhermitian** [comment]
There are some styling issues that need to be resolved


Screen Shot 2020-08-12 at 14 06 00

**Comment by a-matsuo** [comment]
Hi @nonhermitian, I fixed the styling issues. Could you please check it again?

**Pull request merged**